auth/store.go

```diff
@@ -33,16 +33,19 @@ var (
 type AuthStore interface {
 	// AuthEnable() turns on the authentication feature
 	AuthEnable()
+
+	// UpdateBackend updates authStore.be during EtcdServer.applySnapshot()
+	UpdateBackend(b backend.Backend)
```

Comment:
Yes, I misunderstood about the interface of lease. The recovery process is required.